onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1597,5 +1597,200 @@ TEST(InferenceSessionTests, ModelThatTriggersAllocationPlannerToReuseDoubleTenso
   VerifyOutputs(fetches[2].Get<Tensor>(), expected_dims_res3, expected_values_res3);
 }
 
+// The following test is to cover the feature of InferenceSession that allows some session options
+// to flow in from a model file, and use defaults for missing session options/session options not supported for parsing
+// from the model
+static char ort_load_config_from_model_env_var_enabled[] = "ORT_LOAD_CONFIG_FROM_MODEL=1";
+static char ort_load_config_from_model_env_var_disabled[] = "ORT_LOAD_CONFIG_FROM_MODEL=0";
+
+TEST(InferenceSessionTests, LoadModelWithValidOrtConfigJson) {
```

Comment:
can you add a test in onnxruntime_test_python.py to exercise this new feature? 
it would probably require you to expose GetSessionOptions() api in python.

Reply:
Done.